pegged/docs/wiki

```diff
@@ -1 +1 @@
-Subproject commit f111de4928c41cc22d2d093e3362f00bf7f80733
+Subproject commit 69188d442c1e7e4ad3976e5fc3988792723fc097
```

Comment:
I didn't know how github wikis work yet, so I mistakenly had a reference here to a branch in a local clone of your wiki. I don't think that would work, so this is a reference to the latest commit in origin of your wiki instead; I suppose that is correct. Somehow I couldn't revert to what it was before, but that reference was rather old anyway.

I'll amend the PEG-extensions page accordingly online if you merge this.